README.md

```diff
@@ -1,219 +1,178 @@
+  library and enable related features if found. This is for
+  extended-precision computations.
 
-* [Elemental.jl](https://github.com/JuliaParallel/Elemental.jl) is an (in-progress) [Julia](http://julialang.org) interface to Elemental being developed by [Jake Bolewski](https://github.com/jakebolewski), [Jiahao Chen](https://jiahao.github.io), and [Andreas Noack](http://andreasnoack.github.io/academiccv.html).
++ `Hydrogen_ENABLE_QD` (Default: `OFF`): Search for the `QD` library
+  and enable related features if found. This is for extended-precision
+  computations.
 
-* [CVXPY](https://github.com/cvxgrp/cvxpy) is a Python-embedded modeling language for convex optimization problems with an (in-progress) interface to Elemental's distributed Interior Point Methods. This effort is being led by [Steven Diamond](http://web.stanford.edu/~stevend2/).
++ `Hydrogen_ENABLE_MPC` (Default: `OFF`): Search for the GNU MPC
+  library (requires MPFR and GMP as well) and enable related features
+  if found. This is for extended precision.
 
-### Related open-source projects
++ `Hydrogen_USE_CUSTOM_ALLTOALLV` (Default: `OFF`): Avoid
+  MPI_Alltoallv for performance reasons.
 
-**Distributed dense linear algebra**:
++ `Hydrogen_AVOID_COMPLEX_MPI` (Default: `OFF`): Avoid potentially
+  buggy complex MPI routines.
 
-* [ELPA](http://elpa.rzg.mpg.de)
-* [CANDMC](https://github.com/solomonik/CANDMC)
-* [PaRSEC/DPLASMA](http://icl.eecs.utk.edu/projectsdev/parsec/index.html)
-* [PLAPACK](http://www.cs.utexas.edu/~plapack)
-* [ScaLAPACK](http://www.netlib.org/scalapack)
++ `Hydrogen_USE_BYTE_ALLGATHERS` (Default: `OFF`): Avoid BG/P
+  allgather performance bug.
 
-**Distributed sparse-direct linear algebra**:
++ `Hydrogen_CACHE_WARNINGS` (Default: `OFF`): Warns when using
+  cache-unfriendly routines.
 
-* [DSCPACK](http://www.cse.psu.edu/~raghavan/Dscpack/)
-* [MUMPS](http://mumps.enseeiht.fr/)
-* [SuperLU](http://crd-legacy.lbl.gov/~xiaoye/SuperLU/)
++ `Hydrogen_UNALIGNED_WARNINGS` (Default: `OFF`): Warn when performing
+  unaligned redistributions.
 
-**Distributed linear algebra Frameworks**
++ `Hydrogen_VECTOR_WARNINGS` (Default: `OFF`): Warn when vector
+  redistribution chances are missed.
 
-* [PETSc](https://www.mcs.anl.gov/petsc/)
-* [Trilinos](http://trilinos.sandia.gov)
+### Example CMake invocation
 
-**Convex optimization**
+The following builds a CUDA-enabled, CUB-enabled, Aluminum-enabled
+version of Hydrogen:
 
-* [CVXOPT](http://cvxopt.org/)
-* [ECOS](https://github.com/embotech/ecos)
-* [L1-MAGIC](http://users.ece.gatech.edu/~justin/l1magic/)
-* [SDPA](http://sdpa.sourceforge.net/index.html)
+```bash
+    cmake -GNinja \
+        -DCMAKE_BUILD_TYPE=Release \
+        -DBUILD_SHARED_LIBS=ON \
+        -DCMAKE_INSTALL_PREFIX=/path/to/my/install \
+        -DHydrogen_ENABLE_CUDA=ON \
+        -DHydrogen_ENABLE_CUB=ON \
+        -DHydrogen_ENABLE_ALUMINUM=ON \
+        /path/to/hydrogen
```

Comment:
```suggestion
        /path/to/hydrogen
    ninja install
```
- As much as we like `ninja`, might it be easier for users to understand what's going on with `make`?
- I get a CMake error when I copy-pasted this, which I fixed by setting `CUB_INCLUDE_PATH`.  The resulting build disabled Aluminum, so it might be helpful to set `Aluminum_DIR` in this example.